app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala

```diff
@@ -33,6 +33,7 @@ object UserType {
 
   final case class IndividualWithInsufficientConfidenceLevel(
                                                               nino: Option[NINO],
+                                                              sautr: Option[SAUTR],
                                                               name: Name,
                                                               email: Option[Email]
                                                             ) extends UserType
```

Comment:
formatting

Reply:
same here

Reply:
The scalafmt config should force this class to be formatted to something like:

```scala
@Singleton
class BusinessPartnerRecordConnectorImpl @Inject()(
  http: HttpClient,
  val config: ServicesConfig
)(implicit ec: ExecutionContext)
```

ie the class variables are two spaces indented from the very left.

Reply:
oh - not sure why scalafmt is not doing it automatically. Unless we're missing a setting or something. I'll change it